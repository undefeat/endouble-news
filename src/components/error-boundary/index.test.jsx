import React from 'react';
import renderer from 'react-test-renderer';
import ErrorBoundary from '.';

describe('ErrorBoundary', () => {
    it('should render error message', () => {
        const ComponentWithError = () => {
            throw Error();
        };

        const tree = renderer.create(
            <ErrorBoundary>
                <ComponentWithError />
            </ErrorBoundary>,
        );

        expect(tree).toMatchSnapshot();
    });
});
