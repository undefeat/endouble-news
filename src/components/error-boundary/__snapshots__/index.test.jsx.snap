// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBoundary should render error message 1`] = `
<h1>
  OOPS an Error Occurred
</h1>
`;
